### DISCUSSION TITLE
Windows 11 does not automatically update Open WebUI from Docker

### DISCUSSION THREAD

**Discussion by Morphus01** [open-webui/open-webui#2773] [General]
![imagen](https://github.com/open-webui/open-webui/assets/50699709/96e11a2b-f316-414b-97d5-63316f4b8f1f)

# Bug Report
I had problems that the local host with WSL in Linux, did not connect to the ollama url in Windows, so I installed with docker with NVIDIA support, but I have tried to pull the last docker in the cmd:
" docker pull ghcr.io/open-webui/open-webui:latest
latest: Pulling from open-webui/open-webui"

But after a refresh of OpenWebUI stills in 0.125

**Comment by justinh-rahb** [comment]
Just pulling the new image doesn't make Docker use it, you must remove your previous container and recreate it.

```bash
docker rm -f open-webui
<your docker run command again>
```
